src/rgw/driver/rados/rgw_rest_user.cc

```diff
@@ -551,11 +569,20 @@ void RGWOp_Subuser_Modify::execute(optional_yield y)
   rgw_user uid(uid_str);
 
   RESTArgs::get_string(s, "subuser", subuser, &subuser);
-  RESTArgs::get_string(s, "secret-key", secret_key, &secret_key);
+  RESTArgs::get_string(s, "secret", secret_key, &secret_key);
```

Comment:
Though this is out of scope for this PR I noticed the request parameters in the [admin api ops doc](https://docs.ceph.com/en/latest/radosgw/adminops/#id29) says the parameter is `secret` not `secret-key`